### PR TITLE
Bl 518 fe i want to see class details

### PR DESCRIPTION
**Pull request by AnthonyLayne**
## Description

Implemented the designs on Figma (BL-473) to build a basic course list for each program for our /parent-booking route. Built a modal to show an advanced search to narrow down specific courses based on the Parent interest or need. Currently this is only the front-end and the filter and search has no functionality yet. Used .less and Antd design components to style the page and modal.

## Loom Video

https://www.loom.com/share/6abb8af4dd8e41c28db59328e6dcb885

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by Rlauckern — APPROVED** [review]
Goog layout and design it definitely matched the concept of the website. I love your use of ant design and making the forms functional to be updated once that information (Instructors, courses, dates etc...) is ready to be added.

File: src/redux/actions/instructorActions.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ export const getCourses = idToken => async dispatch => {
       type: GET_INSTRUCTOR_COURSES,
       payload: courses,
     });
+    console.log(courses);
```
Comment (dirkknibbe) [inline src/redux/actions/instructorActions.js:58]: Good cleanup 👍

**Review by dirkknibbe — APPROVED** [review]

**Review by EmmS21 — APPROVED** [review]
Layout looks great. Good clean code, well done

**Review by AlexiusThomas — COMMENTED** [review]
great code and the layout looks incredible

**Pull request merged**